game/scripts/vscripts/internal/gamemode.lua

```diff
@@ -172,7 +172,7 @@ function GameMode:_CaptureGameMode()
     --mode:SetCustomHeroMaxLevel ( MAX_LEVEL )
     mode:SetCustomXPRequiredToReachNextLevel( XP_PER_LEVEL_TABLE )
 
-    mode:SetBotThinkingEnabled( USE_STANDARD_DOTA_BOT_THINKING )
+    mode:SetBotThinkingEnabled( true )
```

Comment:
Don't change this here. We have a [setting.lua](https://github.com/OpenAngelArena/oaa/blob/master/game/scripts/vscripts/settings.lua#L32) for that